js/tooltips.js

```diff
@@ -0,0 +1,136 @@
+import Adapt from 'core/js/adapt';
+import _ from 'underscore';
+import logging from './logging';
+import TooltipView from './views/TooltipView';
+import TooltipModel from './models/TooltipModel';
+import Backbone from 'backbone';
+
+class TooltipController extends Backbone.Controller {
+
+  initialize() {
+    _.bindAll(this, 'onMouseOver', 'onMouseOut');
+    this._tooltipData = [];
+    this._containerView = new TooltipContainerView();
+    this._containerView.$el.appendTo('body');
+    this.listenTo(Adapt, 'adapt:preInitialize', this.onAdaptPreInitialize);
+  }
+
+  removeListener() {
+    $(document).off('mouseover', '[data-tooltip-id]', this.onMouseOver);
+  }
+
+  addListener() {
+    this.removeListener();
+    $(document).on('mouseover', '[data-tooltip-id]', this.onMouseOver);
+  }
+
+  getConfig() {
+    return Adapt.course.get('_tooltips');
+  }
+
+  registerGlobalTooltips() {
+    const tooltips = Adapt.course.get('_globals')?._tooltips;
+    if (!tooltips) return;
+    tooltips.forEach(tooltip => this.register(tooltip));
+  }
```

Comment:
@oliverfoster can we maintain `_globals._tooltips` so that, for example, an arbitrary component instance can utilise the tooltip API without requiring change to the underlying plugin?

Reply:
Why not just update the plugin so that it works in the AAT? Not doing that would seem like diverting work from something worthwhile? It's just a tooltip registration call, so not a high barrier?

Do you have any examples where that wouldn't work?